src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -388,8 +413,19 @@ const codeBlockCopyButton = ref<typeof KButton | null>(null)
 const numberOfMatches = ref(0)
 const matchingLineNumbers = ref<number[]>([])
 const currentLineIndex = ref<null | number>(null)
+const codeRef = toRef(props, 'code')
```

Comment:
Since you created a ref from the prop value, I don't think you want it to be changeable, right? 

If that's the case, I'd recommend creating `codeRef` as a `computed` variable since all you need is the value down below.
```suggestion
const codeRef = computed(() => props.code)
```